packages/frontend/src/keybindings.ts

```diff
@@ -94,15 +94,15 @@ export function keyDownEvent2Action(
       // } else if (ev.altKey && ev.code === 'ArrowLeft') {
       // disabled until we find a better keycombination (see https://github.com/deltachat/deltachat-desktop/issues/1796)
       //   return KeybindAction.ChatList_ScrollToSelectedChat
-    } else if (ev.ctrlKey && ev.code === 'KeyF') {
+    } else if (ev.ctrlKey && (ev.key === 'f' || ev.code === 'KeyF')) {
       // https://github.com/deltachat/deltachat-desktop/issues/4579
       if (ev.shiftKey) {
         return KeybindAction.ChatList_SearchInChat
       }
       return KeybindAction.ChatList_FocusSearchInput
-    } else if (ev.ctrlKey && ev.code === 'KeyN') {
+    } else if (ev.ctrlKey && (ev.key === 'n' || ev.code === 'KeyN')) {
       return KeybindAction.NewChat_Open
-    } else if (ev.ctrlKey && ev.code === 'KeyM') {
+    } else if (ev.ctrlKey && (ev.key === 'm' || ev.code === 'KeyM')) {
```

Comment:
Note that with this PR, if there was a layout where 'n' and 'f' have places switched, then pressing either key executes the Search action, and there is no way to execute the Composer_Focus action. 

As I could find out, there is no keyboard where just the three keybindings we have so far make problems, but if you add more keybindings later then at some point there will be problems.

I think it would be better to do:

```
if key == 'f' {
    Search
} else if key == 'n' {
    NewChat
} else if key == 'm' {
    Composer_Focus
} else if code == 'KeyF' {
    Search
} else if code == 'KeyN' {
    NewChat
} else if code == 'KeyM' {
    Composer_Focus
}
```